DESCRIPTION

```diff
@@ -91,6 +91,7 @@ Collate:
     'utilities-path.R'
     'standard-path.R'
     'table-formula.R'
+    'unitConverter.R'
```

Comment:
I am creating a new file for it, only temporarily. 

Once the entire plotting functionality has been implemented, this function (along with its cousins) will be moved to `utilities-plotting.R`.